server.js

```diff
@@ -0,0 +1,29 @@
+const express = require('express')
+const next = require('next')
+
+const port = parseInt(process.env.PORT, 10) || 3000
+const dev = process.env.NODE_ENV !== 'production'
+const app = next({ dev })
+const handle = app.getRequestHandler()
+
+app.prepare()
+.then(() => {
+  const server = express()
+
+  server.get('/', (req, res) => {
+    return app.render(req, res, '/', req.query)
+  })
+
```

Comment:
You'll also need routes for your static pages.
Do you have an articles index view? If so, this will need a route too